spec/support/concerns/project_updater_shared_examples.rb

```diff
@@ -13,21 +13,42 @@
 
   describe '#update_project_etag' do
     let!(:original_project_etag) { project.etag }
-    it {
+    let(:audit_comment) { nil }
+
+    before do
+      subject.audit_comment = audit_comment
       expect(create_subject).to be_truthy
       expect(change_subject).to be_truthy
       subject.update_project_etag
       project.reload
       subject.reload
-      last_project_audit = project.audits.last
-      last_subject_audit = subject.audits.last
+    end
+
+    it {
       expect(project.etag).not_to eq(original_project_etag)
+    }
 
-      expected_comment = last_subject_audit.comment ? last_subject_audit.comment.merge({raised_by_audit: last_subject_audit.id}) : {raised_by_audit: last_subject_audit.id}
-      expect(last_project_audit.comment.symbolize_keys).to eq(expected_comment)
+    describe 'with audit comment' do
```

Comment:
This should be a `context` instead of a `describe`. If the name of the block fits the format of "with ...", then usually it should be a `context`.